pkg/reconciler/events/cloudevent/cloudevent_test.go

```diff
@@ -209,3 +240,62 @@ func TestEventForPipelineRun(t *testing.T) {
 		})
 	}
 }
+
+func TestEventForRun(t *testing.T) {
+	runTests := []struct {
+		desc          string
+		run           *v1alpha1.Run
+		wantEventType TektonEventType
+	}{{
+		desc:          "send a cloud event with unset condition, just started",
+		run:           getRunByCondition("", ""),
+		wantEventType: RunStartedEventV1,
+	}, {
+		desc:          "send a cloud event with unknown status pipelinerun, empty reason",
```

Comment:
should these test names read `with unknown status run`?

Reply:
👍